PyAlbany/PyAlbany/Utils.py

```diff
@@ -0,0 +1,145 @@
+"""@package docstring
```

Comment:
Is it normal that this file is in PyAlbany/PyAlbany/  ?

Reply:
The fact that the innermost folder was named PyAlbany was to ease the `import PyAlbany` in the tests.
I renamed it to PyAlbany/python so that it is clearer now.